<!doctype html>

<script src="https://kit.fontawesome.com/d7cf9532bd.js" crossorigin="anonymous"></script>
<script src="../js/toggle.js"></script>
<html lang="en">
    <head>
        <meta charset="utf-8">
        <meta http-equiv="x-ua-compatible" content="ie=edge">
        <meta name="viewport" content="width=device-width, initial-scale=1">
        <title>Stephan Schiffels - How rare genetic mutations help us understand human history (Part II)</title>
        <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css">
        <!-- <link rel="stylesheet" href="/css/default.css" /> -->
    </head>
    <body>
        <div class="container">
        <header class="section">
            <h1 class="title is-1">
                <a href="../">Stephan Schiffels</a>
            </h1>
            <h2 class="subtitle">
                Population Genetics – Computational Methods - Human History
            </h2>
            <nav class="navbar">
                <div class="navbar-brand">
                    <a role="button" class="navbar-burger" data-target="navMenu" aria-label="menu" aria-expanded="false">
                        <span aria-hidden="true"></span>
                        <span aria-hidden="true"></span>
                        <span aria-hidden="true"></span>
                    </a>
                </div>
                <div class="navbar-menu" id="navMenu">
                    <div class="navbar-start">
                        <a href="../" class="navbar-item">Home</a>
                        <a href="../research.html" class="navbar-item">Research</a>
                        <a href="../publications.html" class="navbar-item">Publications</a>
                        <a href="../talks.html" class="navbar-item">Talks</a>
                        <a href="../videos.html" class="navbar-item">Videos</a>
                        <a href="../resources.html" class="navbar-item">Resources</a>
                        <a href="../blog.html" class="navbar-item">News/Blog</a>
                    </div>
                    <div class="navbar end">
                        <a class="navbar-item" href="https://www.eva.mpg.de/archaeogenetics/staff/stephan-schiffels/"><img src="../images/mpi_logo.jpg"></a>
                        <a class="navbar-item" href="https://twitter.com/stschiff"><span class="icon is-large"><i class="fa-brands fa-twitter fa-2xl"></i></span></a>
                        <a class="navbar-item" href="https://github.com/stschiff"><span class="icon is-large"><i class="fa-brands fa-github fa-2xl"></i></span></a>
                        <a class="navbar-item" href="https://medium.com/stephan-schiffels"><span class="icon is-large"><i class="fa-brands fa-medium fa-2xl"></i></span></a>
                        <a class="navbar-item" href="https://www.researchgate.net/profile/Stephan_Schiffels"><span class="icon is-large"><i class="fa-brands fa-researchgate fa-2xl"></i></span></a>
                        <a class="navbar-item" rel="me" href="https://ecoevo.social/@stschiff"><span class="icon is-large"><i class="fa-brands fa-mastodon fa-2xl"></i></span></a>
                    </div>
                </div>
            </nav>

        </header>

            <div class="columns">
                <div class="column is-two-thirds">
                    <main class="section">
                        <h1 class="title is-2">How rare genetic mutations help us understand human history (Part II)</h1>
                        <div class="content">
                            <div class="mb-5"><a href="../blog.html"><i>← All Blog Posts</i></a></div>
<article>
    <header>
        Posted on May 24, 2017
        
        Categories: blog
    </header>
    
        <img src="../images/rarecoal-schematic.jpg">
    
    <p>In my last blog post I have shown how we estimated the Anglo-Saxon genetic
contribution to modern English from rare variants. Here I’ll show more generally
how we can use rare genetic variants to learn more details about population
history.</p>
<!--more-->
<p>As a motivation, consider this figure from the <a href="http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html">1000 Genomes
paper</a>:</p>
<figure>
<img src="../images/thousand_genomes.jpg" />
<figcaption>
Extended Data Figure 6 from the <a href="http://www.nature.com/nature/journal/v526/n7571/full/nature15393.html#extended-data">1000 Genomes
paper</a>,
showing genotype correlations (upper left) and doubleton sharing counts (lower
right). The latter exhibits much more subtle population structure, due to the
low allele frequency hence the relatively young age of the associated mutations.
</figcaption>
</figure>
<p>The lower right half of this figure shows doubleton sharing counts between all
2,504 individuals in the study. A doubleton is a mutation that is only seen
exactly twice in the entire data set. There are millions of those between any
two people, but they occur a lot more often between individuals within the same
population, or even within the same subpopulation. In particular compared with
the upper left triangle, which shows the genome-similarity of a pair of
individuals, the number of doubletons shows a <em>much</em> finer resolution.</p>
<p>For example, in the case of the GBR population (Great Britain), and the CHS
population (Southern Han Chinese), a sub-population level clustering is visible
in the doubleton allele sharing counts as three small distinct triangles seen in
the plot, revealing three distinct sampling locations in these cohorts. But one
can also see cross-population connections, e.g. between CEU and GBR (which have
very similar genetic ancestry), and between Native Americans (PEL, PUR, MXL and
CLM) and Spanish (IBS), which points to recent European admixture in the
Americas.</p>
<hr />
<p>So apparently, rare genetic mutations indicate fine-scale population structure,
much more so than common variants. In fact, the example above only shows
doubletons; there are many mutations of higher — but still low — allele
frequency which are not only shared among pairs of individuals, but among three
or more samples. We cannot easily visualise their counts in a matrix anymore,
but to give an idea, here is a list with rare allele sharing counts in just four
populations (here Finnish, British, Spanish, Italian, each with about a hundred
individuals) up to allele count 3:</p>
<pre><code>PATTERN   COUNT
(0,0,1,0) 2729828
(0,0,0,1) 2493367
(0,1,0,0) 1368389
(1,0,0,0) 931505
(0,0,2,0) 271082
(0,0,1,1) 256098
(0,0,0,2) 246545
(2,0,0,0) 177262
(0,1,1,0) 140255
(0,1,0,1) 116360
(0,2,0,0) 101485
(3,0,0,0) 74200
(1,0,1,0) 63762
(1,0,0,1) 62110
(1,1,0,0) 60916
(0,0,2,1) 59929
(0,0,1,2) 54368
(0,0,3,0) 51519
(0,0,0,3) 38511
(0,1,1,1) 34817
(0,1,2,0) 28842
(0,1,0,2) 22983
(0,2,1,0) 18414
(1,0,1,1) 15654
(0,2,0,1) 15342
(2,1,0,0) 13555
(2,0,1,0) 13177
(1,1,1,0) 13157
(2,0,0,1) 12512
(1,1,0,1) 12199
(1,0,0,2) 11721
(0,3,0,0) 11183
(1,0,2,0) 10868
(1,2,0,0) 8997</code></pre>
<p>Each row has first the pattern of rare mutations in the four populations (for
example, a pattern of (1,0,1,1) means that there is one observed mutation in
each of the Finnish, Spanish and Italian individuals and none in the British),
and then the number of times that this pattern is observed in the genome. One
can see how the complexity of this <em>joint site frequency spectrum</em> increases
dramatically with the number of populations considered and the maximal allele
count (here only 3, out of hundreds of samples).</p>
<p>Even without modelling, just by looking at the histogram above, there are a
number of qualitative observations we can make. For example, the Finnish
population has the lowest* <em>amount of singletons compared to the other three
populations (931505 compared to 2729828, 2493367 and 1368389), which suggests a
lower effective population size caused for example by a strong bottleneck in the
population history of the Fins. At the same time, of all private variants with
allele count 3, the Finnish have the </em>largest* number compared to the other
populations: 74200 in Finnish, compared to 51519, 38511 and 11183 in Spanish,
Italian and British samples, respectively. This excess of tripletons while
showing a relatively low number of singletons is the signature of a population
contraction combined with isolation, consistent with a recent bottleneck
scenario in Finnish population history (there must have been an expansion after
the bottleneck, but that was so recent that it has not yet left a strong enough
trace in singletons at this resolution). Other observations from this table
include the relatively high degree of allele sharing between Spanish and
Italians at allele counts 2 and 3. This suggests that these two populations are
relatively closely related to each other.</p>
<hr />
<p>One way to explain this pattern of shared variation is via an explicit
population model. We can for example use computer simulations to simulate many
different scenarios of population split times and population sizes and see which
scenario most closely matches the truth. This is the approach taken by other
methods, but it does not scale with so many individuals as considered here.
Moreover, we have found that even only considering rare variants up to allele
count 4, should have enough power to infer population history, but standard
simulation tools always generate <em>all</em> mutations, including common variants,
which is computationally wasteful.</p>
<p>Here is where <em>rarecoal</em> comes in. Rarecoal stands for “Rare Allele Coalescent”
and is a method to efficiently compute the likelihood of rare sharing patterns
under an explicit population model. The key objective with this method is to
compute the expected frequency* *of rare mutation patterns under a given
demographic model. To estimate this frequency (an entry of the joint site
frequency spectrum), rarecoal uses coalescent theory to reconstruct all possible
genealogies in which rare alleles with a particular observed pattern can be
connected with each other.</p>
<figure>
<img src="../images/rarecoal-schematic.jpg" style="width:60.0%" />
<figcaption class="figcaption_hack">
A schematic illustrating the rare allele coalescent, taken from Figure 3 of <a href="https://www.nature.com/articles/ncomms10408">our
paper on Anglo-Saxon history</a>.
</figcaption>
</figure>
<p>All these possible genealogies connecting rare alleles have a particular
property: Their lineages (here shown in red) must all derive from a common
ancestor on which the rare mutation occured. Moreover, this common ancestor has
a common ancestor with all other lineages (in blue) <em>that is older than the time
of the mutation</em>. Any more recent common ancestor between a derived and a
non-derived lineage would require a secondary mutation, which is so unlikely
that it is usually excluded (according to the so-called infinite sites model in
coalescent theory).</p>
<p>It turns out that this condition (all red lineages must have a recent common
ancestor, and that ancestor merges with other lineages at an older time than the
mutation), and some approximations of the coalescent process, are enough to
compute the frequency of any observed pattern of rare mutations very
efficiently. Since this calculation depends on the specifics of the population
model (split times, population sizes and migration rates), we can effectively
compute the likelihood of a model given the data. We can then computationally
find the model whose predicted rare allele frequencies best match the observed
ones, among some family of analysed models.</p>
<p>This model estimation process works remarkably well for simulated data:</p>
<figure>
<img src="../images/rarecoal-sim.jpg" />
<figcaption>
A simulated population history (left), and the inferred model from rarecoal<em>
</em>using rare mutations up to allele count 4. Details can be found in <a href="https://www.nature.com/articles/ncomms10408">Schiffels
et al. 2016</a>
</figcaption>
</figure>
<p>In this simulation, we generated counts of shared rare mutations, very similarly
to the table shown above, given a specific population history (shown on the
left). We then forget the history and use only the data (counts of rare alleles)
to infer the most likely tree-like population model using rarecoal. As can be
seen, the inferred model closely matches the simulated model, demonstrating the
power of the approach.</p>
<p>Finally, we applied our method to the large genomic data set presented in the
Figure and table above, and asked <em>rarecoal </em>to estimate a maximum likelihood
population history model based on rare mutations in this data set. Here is what
we found:</p>
<figure>
<img src="../images/rarecoal_europe_tree.jpg" style="width:80.0%" />
</figure>
<p>First, as we expected from visual inspection of the joint site frequency
spectrum for four of these six populations, the Finnish population has by far
the lowest inferred population size compared to the others. Also, the deepest
split in this tree is between North and Southern European populations. Other
notable aspects are a very large ancestral population size in the branch leading
the Southern European populations, and a close relationship between Netherlands
and South England, presumably associated with the Anglo-Saxon migrations (when
we replace the population from Kent with a sample from Cornwall, we find that
the Cornwall population splits first off the Northern European clade).</p>
<hr />
<p>This was just a short demonstration on how rarecoal works and what it can do.
Since the initial
<a href="https://www.nature.com/mosaic_beta/optout?target=%2Farticles%2Fncomms10408%3FWT.ec_id%3DNCOMMS-20160120%26spJobID%3D842604503%26spMailingID%3D50501936%26spReportId%3DODQyNjA0NTAzS0%26spUserID%3DMTA5NjM3MTAyODYxS0">publication</a>,
I have extended the framework considerably, allowing now to test not only models
varying by population sizes and split times, but also by migration edges. The
newly inferred models are more of an admixture graph than a tree, which is more
appropriate for human history than tree-models. We are preparing a publication
using these new approaches, and there will also be a blog post covering these
new developments soon.</p>
</article>

                        </div>
                    </main>
                </div>
                <div class="column">
                    <aside class="section">
    <h2 class="title is-3">News</h2>
    <ul>
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                    
                </div>
                <div class="column">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>August 16, 2023</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Wang2023-dh.html">
                            <b>High-coverage genome of the Tyrolean Iceman reveals unusually
              high Anatolian farmer ancestry</b>
                        </a> by Ke Wang et al.. <i>Cell Genomics</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Wang2023-dh.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                    
                </div>
                <div class="column">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>May 24, 2023</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Gibbon2023-in.html">
                            <b>Confronting historical legacies of biological anthropology in
              South Africa-Restitution, redress and community-centered
              science: The Sutherland Nine</b>
                        </a> by Victoria E Gibbon et al.. <i>PloS one</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Gibbon2023-in.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                    
                </div>
                <div class="column">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>February 28, 2023</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Schmid2023-um.html">
                            <b>Estimating human mobility in Holocene Western Eurasia with
              large-scale ancient genomic data</b>
                        </a> by Clemens Schmid and <u>Stephan Schiffels</u>. <i>Proceedings of the National Academy of Sciences of the United
              States of America</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Schmid2023-um.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>December 3, 2022</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Wang2022-cv.html">
                            <b>4000-year-old hair from the Middle Nile highlights unusual
              ancient DNA degradation pattern and a potential source of early
              eastern Africa pastoralists</b>
                        </a> by Ke Wang et al.. <i>Scientific reports</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Wang2022-cv.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                    
                </div>
                <div class="column">
                    
                        <img src="//images/cavote.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>November 22, 2022</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Waldman2022-hw.html">
                            <b>Genome-wide data from medieval German Jews show that the
              Ashkenazi founder event pre-dated the 14th century</b>
                        </a> by Shamam Waldman et al.. <i>Cell</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Waldman2022-hw.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>November 5, 2022</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Gretzinger2022-wf.html">
                            <b>Transformations in early medieval England</b>
                        </a> by Joscha Gretzinger and <u>Stephan Schiffels</u>. <i>Current Archaeology</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Gretzinger2022-wf.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                    
                </div>
                <div class="column">
                    
                        <img src="//images/publications/Gretzinger2022-wf.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>November, 2022</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Gretzinger2022-et.html">
                            <b>A glimpse into darkness: Population genetics of the European
               middle ages</b>
                        </a> by Joscha Gretzinger and <u>Stephan Schiffels</u>. <i>New Narratives for the First Millennium AD? : Alte und neue
               Perspektiven der archäologischen Forschung zum 1. Jahrtausend n.
               Chr.</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Gretzinger2022-et.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>November 1, 2022</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Gower2022-px.html">
                            <b>Demes: a standard format for demographic models</b>
                        </a> by Graham Gower et al.. <i>Genetics</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Gower2022-px.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>September 21, 2022</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Gretzinger2022-cg.html">
                            <b>The Anglo-Saxon migration and the formation of the early
               English gene pool</b>
                        </a> by Joscha Gretzinger et al.. <i>Nature</i>.
                    
                    
                </div>
                <div class="column">
                    
                        <img src="../images/publications/Gretzinger2022-cg.jpg">
                    
                </div>
            </li>
            <hr />
        
            <li class="columns">
                <div class="column is-two-thirds">
                    
                        <u>July 1, 2022</u><br />
                        <span class="tag is-success">New Publication</span><br />
                        <a href="../pubs/Schiffels2022-ux.html">
                            <b>Erkenntnisse zur frühmittelalterlichen Migrationsgeschichte
             Nordwesteuropas aus genetischen Analysen</b>
                        </a> by <u>Stephan Schiffels</u> and Joscha Gretzinger. <i>Berichte aus dem Landesmuseum Natur und Mensch Oldenburg</i>.
                    
                    
                </div>
                <div class="column">
                    
                </div>
            </li>
            <hr />
        
    </ul>
</aside>
                </div>
            </div>

            <footer class="footer has-text-centered">
                Powered by
                <a href="http://jaspervdj.be/hakyll">Hakyll</a> and <a href="https://bulma.io">Bulma</a>. Sourcecode available on
                <a href="https://github.com/stschiff/stschiff.github.io">github</a>.
            </footer>
        </div>
    </div>
    </body>
</html>
