---
title: Home
menu_Home: True
---

I am group leader for population genetics at the Department for
Archaeogenetics of the [Max Planck Institute for the Science of Human
History in Jena](http://www.shh.mpg.de/en).
My [group](http://stephanschiffels.de/?page_id=8) at the MPI
investigates human history by means of Genetics, and develops
computational methods to model and analyse ancient and modern genomic
data.

## Computational Methods for Genomic Analyses

![](images/population-inference-methods-images.png){width="100%"} An important part of our work consists of developing new
methods to analyse genomic data in order to statistically infer details
about the underlying evolutionary dynamics. At the heart of this
methodological work is the growing amount of data available in the field
of population genomics. When in 2010 there were only a handful of high
quality human genomes completely sequenced, nowadays it is thousands and
growing. The information that this data carries about our past
demography, population diversification, and adaptation processes is
vast, in principle. However, extracting this information out of genomes
requires novel methods that are both based on sound foundations of
population genetic theory and efficient enough to cope with the *Big
Data *that the field of Genetics has become. Furthermore, a relatively
novel requirement of such methods is the integration of ancient DNA,
which is becoming increasingly available. Important examples of methods
that I have developed in recent years and which are further developed in
my group include [MSMC](http://www.github.com/stschiff/msmc) (published
in [Schiffels and Durbin,
2014](https://www.nature.com/articles/ng.3015)) and
[rarecoal](http://www.github.com/stschiff/rarecoal) (published in
[Schiffels et al. 2016](https://www.nature.com/articles/ncomms10408) and
[Flegontov et al.
2017](https://www.biorxiv.org/content/early/2017/10/13/203018)\]). We
are also very active in developing bioinformatic tools and pipelines to
help with processing sequencing data from ancient DNA. Examples for such
tools include the [sequenceTools
package](https://github.com/stschiff/sequenceTools), the
[mergeAndClipFastq](https://github.com/stschiff/mergeAndClipFastq/tree/master)
program, and various processing tools in the
[rarecoal-tools](https://github.com/stschiff/rarecoal-tools) package.

## Human History through genetics

![](images/historical-genetics-images.png){width="100%"} One
important goal of genomic analysis is to reconstruct human history by
mean of genetics. Up until a few years ago, this was an endeavour that
was mostly based on modern genomic data that is became increasingly
available. As a major game changer, we are now able to not only
indirectly look into the past through our population genetic models
based on modern data, but directly through analysing ancient DNA. This
relatively new field of \"Genetic history\" has lead to new insights
into human history from Genetics in recent years, including on migration
movements between regions and continents, the separation of people into
spatially separated subpopulations, as well as changes in population
size. Concrete examples of our research covers continental peopling
events, for example into America (see [Raghavan et al.
2015](http://science.sciencemag.org/content/349/6250/aab3884) and
[Flegontov et al.
2017](https://www.biorxiv.org/content/early/2017/10/13/203018)),
Australia (see [Malaspinas et al.
2016](https://www.nature.com/articles/nature18299)), the early
Anglo-Saxon migrations into Britain (see [Schiffels et al.
2016](https://www.nature.com/articles/ncomms10408) and two blog posts
[here](http://stephanschiffels.de/?p=82) and
[here](http://stephanschiffels.de/?p=215)), or insights into population
changes in [Egypt](https://www.nature.com/articles/ncomms15694) within
the last 2,000 years. Our work has also been picked up by the press, for
example by the
[BBC](http://www.bbc.com/news/science-environment-35344663) or the
[Washington
Post](https://www.washingtonpost.com/news/speaking-of-science/wp/2017/05/30/dna-from-ancient-egyptian-mummies-reveals-their-ancestry/?utm_term=.b647d2170fa6).

## Population Genetic Theory

![](images/theoretical-population-genetics-images.png){width="100%"} I received my PhD from the Institute for Theoretical
Physics in Cologne, and my
[dissertation](http://kups.ub.uni-koeln.de/4795/) was in population
genetic theory. Since then I have worked on a number of studies that
explore evolutionary dynamics of adaptation, in particular in low
recombining organisms or genomic regions. Examples for such studies
include the effect of clonal interference on fixation probabilities of
selected mutations (see [Schiffels et al.
2011](http://www.genetics.org/content/189/4/1361)), how quantitative
trait equilibria affect the the segregations of mutations at a genomic
level (see [Nourmohammad et al.
2012](http://iopscience.iop.org/article/10.1088/1742-5468/2013/01/P01012/meta)),
and analyses of adaptive evolution under strong linkage in natural
populations of *Drosophila melanogaster* ([Schiffels et al.
2017)](https://www.biorxiv.org/content/early/2017/11/29/226670).

